int32 x
int32 y
int32 width
int32 height
float32 score
int32 area
Keypoint keypoint
